pymc_marketing/product_incrementality/mv_its.py

```diff
@@ -0,0 +1,573 @@
+#   Copyright 2024 The PyMC Labs Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""Multivariate Interrupted Time Series Analysis for Product Incrementality."""
+
+import json
+from typing import Any
+
+import arviz as az
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mtick
+import numpy as np
+import pandas as pd
+import pymc as pm
+from typing_extensions import Self
+
+from pymc_marketing.model_builder import ModelBuilder
+from pymc_marketing.prior import Prior
+
+HDI_ALPHA = 0.5
+
+
+class MVITS(ModelBuilder):
+    """Multivariate Interrupted Time Series class.
+
+    Class to perform a multivariate interrupted time series analysis with the
+    specific intent of determining where the sales of a new product came from.
+    """
+
+    _model_type = "Multivariate Interrupted Time Series"
+    version = "0.1.0"
+
+    def __init__(
+        self,
+        existing_sales: list[str],
+        market_saturated: bool = True,
```

Comment:
I find `market_saturated` reads a bit weird. What are your thoughts on `saturated_market` instead? ie `MVITS(..., saturated_market=True)`

Reply:
I'd be happy with `saturated_market`

Reply:
Done in https://github.com/pymc-labs/pymc-marketing/pull/803/commits/c96a56803a769e17ee77dcbc3d206b938634a83a